src/signals/incident-management/containers/IncidentDetail/IncidentDetail.tsx

```diff
@@ -184,13 +184,28 @@ const IncidentDetail = () => {
   }, [attachments])
 
   useEffect(() => {
-    if (!incident?.category) return
+    if (!incident?.category || !subcategories) return
 
-    const { main_slug, sub_slug } = incident.category
-    getDefaultTexts(
-      `${configuration.TERMS_ENDPOINT}${main_slug}/sub_categories/${sub_slug}/status-message-templates`
-    )
-  }, [incident?.category, getDefaultTexts])
+    if (configuration.featureFlags.showStandardTextAdminV1) {
```

Comment:
why independent ifs and not in structure of if -> else ifs?

Reply:
Personally I find this more readable.